health/notifications/alarm-notify.sh.in

```diff
@@ -1871,7 +1864,7 @@ send_awssns() {
             info "sent Amazon SNS notification for: ${host} ${chart}.${name} is ${status} to '${target}'"
             sent=$((sent + 1))
         else
-            error "failed to send Amazon SNS notification for: ${host} ${chart}.${name} is ${status} to '${target}'" 
+            error "failed to send Amazon SNS notification for: ${host} ${chart}.${name} is ${status} to '${target}'"
```

Comment:
This type of cleanup is probably a good thing, but it is unrelated to the PR.  At minimum, it should ideally be an independent commit.

Reply:
I understand. The whitespace removal was done automatically by my editor. Do you want me to try to revert these changes?

Reply:
I'm not sure, lets see what @ktsaou says about it, he may just not care.

Reply:
Personally I don't care :) If it was some larger PR with many such lines changed then I would care, but for one line - nah.

Reply:
There are actually some changes regarding whitespace, but I think it is pretty much straightforward to review these. But let me know.